CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
+
+### Web3
+
+- Implemented “Select a token to send” view which allows selecting an account and an asset at the same time. ([#7523](https://github.com/brave/brave-ios/issues/7523))
+- Removed network name from the “Public Encryption Key Request” panel. ([#7495](https://github.com/brave/brave-ios/issues/7495))
+- Improved NFT importing in Brave Wallet by introducing a new button to add custom assets quicker in NFT tab view and also simplifying the “Add Custom Asset” form by moving moving “Token address” field to the top. ([#7556](https://github.com/brave/brave-ios/issues/7556))
+- Updated “Select a Token to Send” screen to sort test network tokens to be displayed at the end. ([#7640](https://github.com/brave/brave-ios/issues/7640))
+- Updated network filters to allow multiple networks to be selected. ([#7607](https://github.com/brave/brave-ios/issues/7607))
+- Updated the “Secondary Networks” list in the “Select Network Filter” screen by moving “Filecoin EVM Testnet” to be included with the other testnets. ([#7583](https://github.com/brave/brave-ios/issues/7583))
+- Resolved migrating user assets when restoring from the wallet lock screen. ([#7785](https://github.com/brave/brave-ios/issues/7785))
+- Resolved asset preferences from being overwritten when auto-discovery returns results. ([#7443](https://github.com/brave/brave-ios/issues/7443))
+- Fixed auto-discovered assets from being duplicated after restoring Brave Wallet. ([#7804](https://github.com/brave/brave-ios/issues/7804))
+- Fixed “Send” screen in Wallet to retain the last selected token when opened or display the native token by default. ([#7759](https://github.com/brave/brave-ios/issues/7759))
+- Fixed native assets to be listed when manually adding custom network. ([#7623](https://github.com/brave/brave-ios/issues/7623))
+- Fixed Wallet panel from being closed also when user dismisses the pending quest panel. ([#7505](https://github.com/brave/brave-ios/issues/7505))
+
+### General  
+
+- Implemented bookmark count feature for bookmark folders. ([#7674](https://github.com/brave/brave-ios/issues/7674))
+- Added support for using custom search engine in Private Mode. ([#7733](https://github.com/brave/brave-ios/issues/7733))
+- Added “Media” settings menu in main settings. [#7677](https://github.com/brave/brave-ios/issues/7677)
+- Added “Highest Quality Playback” preference in “Media” setting for YouTube video resolution. ([#7055](https://github.com/brave/brave-ios/issues/7055))
+- Added voice search feature in the URL bar. ([#7032](https://github.com/brave/brave-ios/issues/7032))
+- Added new search engines for South Korea region. ([#6912](https://github.com/brave/brave-ios/issues/6912))
+- Added support for using component updater to fetch background and sponsored images. ([#7467](https://github.com/brave/brave-ios/issues/7467))
+- Added Brave Talk Jitsi SDK Integration to release channel builds. ([#7560](https://github.com/brave/brave-ios/issues/7560))
```

Comment:
I don't think we need to capitalize `Integration` in this instance. Probably don't need to mention the release channel as it's a given as these notes are for releases on the `release` channel. Maybe something like this?

```
- Added Jitsi SDK integration for Brave Talk. ([#7560](https://github.com/brave/brave-ios/issues/7560))
```

Reply:
Very good, thanks.